### DISCUSSION TITLE
does it work with `pages` directory?

### DISCUSSION THREAD

**Discussion by TheMikeyRoss** [avitorio/outstatic#161] [Q&A] [answered]
I noticed in the docs it only mentions the `app` directory. 

Does it work with `pages` directory?

**Comment by avitorio** [comment, marked as answer]
Hey, no it only works with the app directory at the moment. But you can run both /pages and /app in parallel. So if you are on Next.js > 13 you should be able to use it. If you are on Next.js 12, you'll have to use Outstatic < 1.0.0

**Reply by TheMikeyRoss** [reply]
I'm actually using nextjs 14 but didn't mentally migrate to the `app` directory. I'm still personally attached to the `pages` directory 😅

So I create a folder called `app` adjacent to `pages` folder? and inside the `app` folder I just follow the docs?

**Reply by avitorio** [reply]
Yup, that should do the trick 😂
Let me know if it works!
